waspc/src/Wasp/Psl/Ast/Enum.hs

```diff
@@ -0,0 +1,24 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+
+module Wasp.Psl.Ast.Enum
+  ( Enum (..),
+    Element (..),
+  )
+where
+
+import Data.Data (Data)
+import Wasp.Psl.Ast.Attribute (Attribute)
+import Prelude hiding (Enum)
+
+type Name = String
```

Comment:
You are repating this `type Name` -> you should rather have it just in one place and export it.